build.gradle

```diff
@@ -2,7 +2,7 @@
 
 buildscript {
     repositories {
-        mavenCentral()
+        jcenter()
```

Comment:
Thank you for your reply!
There is no problem in that way. But what is the reason to leave the `mavenCentral`?
In other google samples do not seem to use only `jcenter`.
Is there a difference between the contents of `mavenCentral` and `jcenter`? :open_mouth: 

refer to
https://github.com/googlesamples/android-UniversalMusicPlayer/blob/master/build.gradle
https://github.com/googlesamples/android-topeka/blob/master/build.gradle